<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qtCMBBathymetryDialog</class>
 <widget class="QDialog" name="qtCMBBathymetryDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>522</width>
    <height>246</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bathymetry Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" colspan="3">
    <layout class="QHBoxLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Bathymetry Source</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="BathymetrySources"/>
     </item>
     <item>
      <widget class="QPushButton" name="FileBrowserButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Browse ...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QGroupBox" name="ParametersGroup">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Parameters</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Radius for Averaging Elevation:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="ElevationRadius"/>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="checkBoxHighLimit">
        <property name="text">
         <string>Set Highest Elevation</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" rowspan="2">
       <widget class="QLineEdit" name="highElevation">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0" rowspan="2">
       <widget class="QCheckBox" name="checkBoxLowLimit">
        <property name="text">
         <string>Set Lowest Elevation</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lowElevation">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="RemoveBathymetryButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Remove Bathymetry</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>162</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QCheckBox" name="OnlyApplyToVisibleMesh">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Apply bathymetry only to the visible meshes on the model</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>qtCMBBathymetryDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>510</x>
     <y>234</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>245</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>qtCMBBathymetryDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>510</x>
     <y>234</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>245</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxHighLimit</sender>
   <signal>toggled(bool)</signal>
   <receiver>highElevation</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>127</x>
     <y>116</y>
    </hint>
    <hint type="destinationlabel">
     <x>260</x>
     <y>118</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxLowLimit</sender>
   <signal>toggled(bool)</signal>
   <receiver>lowElevation</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>94</x>
     <y>147</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>154</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
